lib/functions/buildFileList.sh

```diff
@@ -452,6 +451,7 @@ function BuildFileList() {
       # Append the file to the array #
       ##############################p##
       FILE_ARRAY_HTML+=("${FILE}")
+      FILE_ARRAY_SEMGREP+=("${FILE}")
```

Comment:
Are you sure [r2c-security-audit](https://semgrep.dev/p/r2c-security-audit) supports HTML?

Template files can have html extensions, but I'm not sure if semgrep actually supports or looks at those. When I search for 'django' or 'html' in their search bar, nothing indicates that semgrep can actually handle template files.

Reply:
Resolved offline; it's under their 'generic' section (along with a rule for dockerfiles!)